### ISSUE TITLE
Paging is broken at build because reference json files have encoded file names

### ISSUE THREAD

**Issue by ArthurClemens** [phenomic/phenomic#1158]
File name
```
after-MjAxNy0wMS0wMS1lZXJzdGUtcG9zdA%3D%3D.json
```
should be
```
after-MjAxNy0wMS0wMS1lZXJzdGUtcG9zdA==.json
```

**Comment by DominikGuzei** [comment]
I think this is also the problem why it fails to build on Netlify, here is the deploy output:
```
6:21:52 PM: Build started
6:21:52 PM: Fetching cached dependencies
6:21:52 PM: Starting to download cache of 132.0MB
6:21:54 PM: Finished downloading cache in 1.121481141s
6:22:00 PM: Preparing Git Reference refs/heads/chore/refactor-to-phenomic
6:22:02 PM: Running build command: npm run build
6:22:05 PM: Using node version '8.2.1' from .nvmrc
6:22:06 PM: v8.2.1 is already installed.
6:22:07 PM: Now using node v8.2.1 (npm v5.3.0)
6:22:07 PM: Using version v8.2.1 of node
6:22:07 PM: Using /opt/buildhome/.rvm/gems/ruby-2.1.2
6:22:09 PM: Installing NPM modules using Yarn version 0.18.1
6:22:12 PM: yarn install v0.18.1
6:22:12 PM: [1/4] Resolving packages...
6:22:13 PM: [2/4] Fetching packages...
6:22:28 PM: warning fsevents@1.1.2: The platform "linux" is incompatible with this module.
6:22:28 PM: info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
6:22:28 PM: [3/4] Linking dependencies...
6:22:28 PM: warning Unmet peer dependency "redux@^2.0.0 || ^3.0.0".
6:22:28 PM: warning Incorrect peer dependency "react@^0.14 || ~15.4.0".
6:22:28 PM: warning Unmet peer dependency "react-addons-css-transition-group@^0.14.0 || ~15.4.0".
6:22:28 PM: warning Incorrect peer dependency "react-dom@^0.14.0 || ~15.4.0".
6:22:29 PM: warning Unmet peer dependency "prop-types@^15.5.10".
6:22:29 PM: warning Unmet peer dependency "lodash@^4.17.2".
6:22:29 PM: warning Unmet peer dependency "webpack@^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0".
6:22:29 PM: warning Unmet peer dependency "file-loader@*".
6:22:29 PM: warning Unmet peer dependency "babel-core@6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc".
6:22:47 PM: [4/4] Building fresh packages...
6:22:51 PM: Done in 39.04s.
6:22:51 PM: NPM modules installed using Yarn
6:22:52 PM: 
> josoor-frontend@0.0.1 build /opt/build/repo
> phenomic build
6:22:55 PM: Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
6:22:56 PM: Warning: Accessing createClass via the main React package is deprecated, and will be removed in React v16.0. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class v15.* is available on npm as a temporary, drop-in replacement. For more info see https://fb.me/react-create-class
6:22:56 PM: ⚡️ Hey! Let's get on with it
6:23:09 PM: 📦 Webpack client build done 14952ms
6:23:14 PM: Warning: %s 9:12-34 Critical dependency: the request of a dependency is an expression
6:23:14 PM: 📦 Webpack static build done 5841ms
6:23:16 PM: 📝 Content processed 1428ms
6:23:16 PM: 📃 Pre-rendering finished 253ms
6:23:16 PM: Cached NPM modules
6:23:16 PM: Saved Yarn cache
6:23:16 PM: Build complete: exit code: 0
6:23:17 PM: Cleaning up docker container
6:23:17 PM: Starting to deploy site from 'dist/'
6:23:17 PM: Deploying to CDN
6:23:17 PM: Uploading 11 files
6:23:17 PM: Uploading file blog/after/MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU=/index.html
6:23:17 PM: Uploading file phenomic.main.js
6:23:17 PM: Uploading file impress/index.html
6:23:17 PM: Uploading file phenomic/posts/item/2017-09-12-welcome.json
6:23:17 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:17 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2.json
6:23:17 PM: Uploading file blog/index.html
6:23:17 PM: Uploading file blog/posts/2017-09-12-welcome/index.html
6:23:17 PM: Uploading file index.html
6:23:17 PM: Uploading file images/refugees.png
6:23:18 PM: Uploading file styles.css
6:23:19 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:19 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:20 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:20 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:21 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:22 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:23 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:25 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:25 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:27 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:27 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:30 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:30 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:35 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:36 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:45 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:45 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:56 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:23:57 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:24:19 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:24:20 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:24:44 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:24:45 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:25:17 PM: Uploading file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:25:17 PM: Failed to upload file phenomic/posts/by-default/1/desc/limit-2/after-MjAxNy0wOC0wMi0yMDE3LTA5LTEyLXdlbGNvbWU%3D.json
6:25:18 PM: Failed to deploy dist/ to CDN
6:25:18 PM: [PUT /deploys/{deploy_id}/files/{path}][500] uploadDeployFile default &{Code:0 Message:<nil>}
6:25:20 PM: Finished processing build request in 3m28.173336883s
```

**Comment by MoOx** [comment]
Fixed in [v1.0.0-alpha.7](https://github.com/phenomic/phenomic/releases/tag/v1.0.0-alpha.7)
